staging/src/k8s.io/kubelet/pkg/apis/podresources/v1alpha1/api.proto

```diff
@@ -39,10 +39,22 @@ message PodResources {
 message ContainerResources {
     string name = 1;
     repeated ContainerDevices devices = 2;
+    repeated uint32 cpu_ids = 3;
```

Comment:
plase note it is int64 in the KEP: https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2043-pod-resource-concrete-assigments#proposed-api

which version is correct? this or KEP's?

Reply:
right, int32 was in initial version, but since kernel keeps it as int, it's more safer to keep it as int64. I'll update it soon.